### PR TITLE
better error messages in bem scripts

### PR DESCRIPTION
**Pull request by agramfort**

### CODE REVIEW

File: mne/bem.py [hunk 1]
```diff
@@ -1346,11 +1340,21 @@ def write_bem_solution(fname, bem):
...
-def _prepare_env(subject, subjects_dir):
+def _prepare_env(subject, subjects_dir, requires_freesurfer, requires_mne):
     """Helper to prepare an env object for subprocess calls"""
     env = os.environ.copy()
+    if requires_freesurfer and not os.environ.get('FREESURFER_HOME'):
+        raise RuntimeError('FREESURFER_HOME environment variable not set. '
+                           'freesurfer is not setup.')
+    if requires_mne and not os.environ.get('MNE_ROOT'):
+        raise RuntimeError('MNE_ROOT environment variable not set. '
+                           'mne is not setup.')
```
Comment (dengemann) [inline mne/bem.py]: A bit redundant messages? Maybe say "some functions require this variable to be set." or so. Another options is to remove the last phrase.

Comment (agramfort) [inline mne/bem.py]: I wrote it this way as it says how the problem was found and how to fix it.
By calling mne_setup

Comment (dengemann) [inline mne/bem.py]: But you repeat yourself and the second phrases are not very informative.

Comment (dengemann) [inline mne/bem.py]: 'I cannot find freesurfer, the FREESURFER_HOME environment variable not set.'

'I cannot find the MNE command line tools, the MNE_ROOT environment variable not set. '

Comment (agramfort) [inline mne/bem.py]: done

**Comment by dengemann** [comment]
Thanks!

**Pull request merged**